frontend/src/components/labwork/step/LabworkStep.tsx

```diff
@@ -123,6 +126,38 @@ const LabworkStep = ({ protocol, step, stepSamples }: LabworkStepPageProps) => {
 			}
 		}
 		, [step, selectedTemplate, navigate, dispatch])
+  
+  const handleExecuteAutomation = useCallback(
+    async () => {
+      try {
+        const response = await dispatch(requestAutomationExecution(step.id))
+        if (response) {
+          const success = response.data.result.success
+          if (success) {
+            dispatch(flushSamplesAtStep(step.id))
+            const AUTOMATION_SUCCESS_NOTIFICATION_KEY = `LabworkStep.automation-success-${step.id}`
```

Comment:
suggestion to store these somewhere as a constant

Reply:
Yeah i think we can use python formating to make it a constant.

Reply:
Looking at how the value is variable it would not make much sense to move part of it outside you still would have to check here then also somewhere else. Maybe i am not getting your meaning.